platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileListCommand.java

```diff
@@ -60,9 +61,7 @@ private void listProfileFiles() {
       files
           .filter(Files::isRegularFile)
           .filter(file -> file.toAbsolutePath().toString().endsWith(PROFILE_EXTENSION))
-          .forEach(
-              profile ->
-                  console.println(profile.toFile().getName().replaceAll(PROFILE_EXTENSION, "")));
+          .forEach(profile -> console.println(FilenameUtils.removeExtension(profile.toString())));
```

Comment:
❗️  I do not think we should start printing out the whole path on the console now? It used to only put the name part (minus the extension). 
👍  on using `FilenameUtils.removeExtension()`